tests/main/snap-system-key/task.yaml

```diff
@@ -1,5 +1,10 @@
 summary: Ensure security profile re-generation works with system-key
 
+details: |
+    Check that security profile are re-generated when a system-key
+    version mismatch is detected. Also check that `snap run` waits
+    for system-key updates unless when it can talk to snapd.
```

Comment:
```suggestion
    Check that security profiles are re-generated when a system-key
    version mismatch is detected. Also check that `snap run` waits
    for system-key updates unless it can talk to snapd.
```